exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/metadata/MetadataPathUtils.java

```diff
@@ -60,6 +61,22 @@
    * @param baseDir base parent directory
    * @return list of files with absolute paths
    */
+  public static List<ParquetFileMetadata_v4> convertToFilesWithAbsolutePathsForV4(
```

Comment:
Please modify existing method `convertToFilesWithAbsolutePaths` for using with `ParquetFileMetadata_v4` and `ParquetFileMetadata_v3` to avoid duplicating the code.

Reply:
Changed the method.